src/mark.ts

```diff
@@ -1,36 +1,35 @@
-/* @flow */
-
 import * as React from 'react';
 
 import markRegExp from './markRegExp';
 import markTerm from './markTerm';
+import type {TagFunction} from './createMarker';
```

Comment:
It seems TS is fine with that, but there's a cyclic import here: `mark` imports `createMarker` for this type, and `createMarker` imports `mark` for its default export. Maybe we should put the `TagFunction` type in a separate file? Maybe a `types.ts` file?

Reply:
Sounds good, I had the same concern